recipe/meta.yaml

```
@@ -1,38 +1,53 @@
 {% set name = "bson" %}
-{% set version = "0.5.9" %}
-{% set sha256 = "84a06dd1fa350d5814d9845abad982e8ac4c9768c4f41afeaa5fdb6d58570158" %}
+{% set version = "0.5.10" %}
+{% set sha256 = "fef2a86682570c3622519ddafb7477e9b876d440eba4e04017b1a5f3b49ccc27" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/py-bson/bson/archive/refs/tags/{{version}}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - setuptools
     - pip
+    - wheel
   run:
     - python
     - six
     - python-dateutil >=2.4.0
 
 test:
+  source_files:
+    - bson/tests
   imports:
     - bson
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
+    - pytest -v  # [py<312]
+    # These test files use assertEquals, removed in python 3.12.
+    - pytest -v --ignore="bson/tests/test_array.py" --ignore="bson/tests/test_random_tree.py" # [py>=312]
 
 about:
-  summary: Independent BSON codec for Python that doesn't depend on MongoDB.
   home: https://github.com/py-bson/bson
+  dev_url: https://github.com/py-bson/bson
+  doc_url: https://github.com/py-bson/bson/blob/master/README.rst
+  summary: Independent BSON codec for Python that doesn't depend on MongoDB.
+  description: |
+    Independent BSON codec for Python that doesn't depend on MongoDB. 
+    The bson ObjectId implementation is forked from the PyMongo project,
+    licensed under the Version 2.0 Apache License.
   license: BSD-3-Clause
   license_family: BSD
   license_file: LICENSE.txt
```

```suggestion
  license: Apache-2.0
  license_family: Apache
  license_file: LICENSE_APACHE
```
Not sure why no errors were detected but it seems like they updated the license to Apache [here](https://github.com/py-bson/bson/blob/0.5.10/LICENSE_APACHE)

---

@Jrice1317 It's unclear which one wins (the metadata or the license). We will have to check.

---

@JeanChristopheMorinPerso There are actually license files for both. I can list both license files, but I'm not sure what to put in license_family. `OTHER` perhaps?
- BSD-3-Clause: https://github.com/py-bson/bson/blob/master/LICENSE
- Apache-2.0: https://github.com/py-bson/bson/blob/master/LICENSE_APACHE

---

@ViridianMelody something similar to this:
https://github.com/AnacondaRecipes/typed-ast-feedstock/blob/6ba74e5ecba70789babfa440364c5c473862071b/recipe/meta.yaml#L39